apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -124,7 +124,7 @@ export class SendMessageEmail extends SendMessageBase {
       return;
     }
 
-    const overrides: Record<string, any> = command.overrides[integration?.providerId] || {};
```

Comment:
Previously this was the actual `overrides` object here, instead of the global overrides. Maybe we can optionally merge both providerId overrides and `email` overrides to a unified override? 

The providerId should overtak the "email" once.

And then we can use this object instead of command.overrides downstream. Wdyt?

Reply:
I think we could, one sec i will add the additional provider to it.